.gitignore

```diff
@@ -11,3 +11,5 @@
 /log/*
 !/log/.keep
 /tmp
+
+/npm-debug.log
```

Comment:
@Jpease1020 Check out the top response here:

http://stackoverflow.com/questions/1139762/ignore-files-that-have-already-been-committed-to-a-git-repository